src/Aspire.Cli/Resources/ExecCommandStrings.resx

```diff
@@ -118,21 +118,30 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Description" xml:space="preserve">
-    <value>Run an Aspire app host to execute a command against the resource. (Preview)</value>
+    <value>Execute commands in the context of an Aspire application resource. Starts the AppHost, waits for resources to initialize, then runs the specified command in the target resource's environment. (Preview)
+
+Examples:
+  aspire exec --resource api dotnet build
+  aspire exec --resource api -- dotnet test --logger console
+  aspire exec --start-resource worker pwsh -c "Get-Process"
```

Comment:
Is this missing `--`?

Reply:
not really, you can supply a command without separator if you want to. `ExecCommand` parses the unmatched tokens into command and that's why that works